File: C/Program Mencetak Barisan Bilangan Ganjil Genap.c
#include <stdio.h>

int main() {
    int n;

    printf("Program Mencetak Barisan Bilangan Ganjil Genap\n");
    printf("visit us hobingoding.com\n\n");
    // Input Nilai
    printf("Input Nilai n : ");
    scanf("%d", &n);

    // Barisan Bilangan Genap
    printf("Bilangan Genap : ");
    for(int i = 0; i <= n; i++) {
        if(i % 2 == 0)
            printf("%d ", i);
    }

    // Barisan Bilangan Ganjil
    printf("\n");
    printf("Bilangan Ganjil : ");
    for(int i = 0; i <= n; i++) {
        if(i % 2 != 0)
            printf("%d ", i);
    }
    return 0;
}

File: Python/Program Mencetak Barisan Bilangan Ganjil Genap.py
print("Program Mencetak Barisan Bilangan Ganjil Genap")
print("visit us hobingoding.com")
print()
# Input Nilai n
n = int(input("Input Nilai n : "))

# Barisan Bilangan Genap
print("Bilangan Genap :", end=" ")
for i in range(0, n+1, 2):
    print(i, end=" ")
    
# Barisan Bilangan Ganjil
print()
print("Bilangan Ganjil :", end=" ")
for i in range(1, n+1, 2):
    print(i, end=" ")